hydra/_internal/callback.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
```

Comment:
I guess we can move this to hydra.experimental instead of hydra._internal.
we can followup with documentation later.